doc/mgr/dashboard.rst

```diff
@@ -178,6 +178,58 @@ exist and you may find yourself in the situation that you have to use them::
   $ ceph dashboard set-rgw-api-admin-resource <admin-resource>
   $ ceph dashboard set-rgw-api-user-id <user-id>
 
+Configuration Grafana to be used by the Frontend
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Grafana and Prometheus are likely going to be bundled and installed by some
+orchestration tools along Ceph in the near future, but currently, you will have
+to install and configure both manually. After you have installed Prometheus and
+Grafana on your preferred hosts, proceed with the following steps::
+
+ 1. Enable the Ceph Exporter which comes as Ceph Manager module by running::
+
+    $ ceph mgr module enable prometheus
+
+ 2. Add the corresponding scrape configuration to Prometheus. This may look
+    like::
+
+        global:
+          scrape_interval: 5s
+
+        scrape_configs:
+          - job_name: 'prometheus'
+            static_configs:
+              - targets: ['localhost:9090']
+          - job_name: 'ceph'
+            static_configs:
+              - targets: ['localhost:9283']
+          - job_name: 'node-exporter'
+            static_configs:
+              - targets: ['localhost:9100']
+
+ 3. Add Prometheus as data source to Grafana
+
+ 4. Install the `vonage-status-panel` plugin using::
+
+        grafana-cli plugins install vonage-status-panel
+
+ 4. Add the Dashboards to Grafana by importing them
+
+ 5. Configure Grafana in `/etc/grafana/grafana.ini` to adapt the URLs to the
+    Ceph Dashboard properly::
+
+        root_url = http://localhost:3000/api/grafana/proxy
+
+After you have configured Grafana and Prometheus, you will need to tell the
+Ceph Manager Dashboard where it can access Grafana and what the credentials are
+to do so. This can be done by using the following commands::
+
+  $ ceph dashboard set-grafana-api-host <host> # Default: 'localhost'
+  $ ceph dashboard set-grafana-api-port <port> # Default: 3000
+  $ ceph dashboard set-grafana-api-scheme <scheme> # Default: 'http'
```

Comment:
hm. would it make sense to unify this to a single url? `http://host:port`?

Reply:
I agree, that would significantly simplify the configuration as one could simply cut and paste the entire URL without having to break it up in components.

Reply:
Makes sense